embark/uploader/views.py

```diff
@@ -20,4 +20,11 @@ def upload_file(request):
     """
     Uploading method
     """
-    pass
+    html_body = get_template('uploader/index.html')
+    return HttpResponse(html_body.render())
+
+def save_file(request):
+    if request.method == 'POST':
+        print(request.POST['FILE'])
```

Comment:
-  Use `crsf_exempt` header. 
- Restrict this view to only post requests(@require_http_methods(["POST"]))
- Access `request.FILES` directly
- Use `FileSystemStorage` to save the files

Reply:
Ashutoshs suggestions seem reasonable.